german.html

```diff
@@ -0,0 +1,115 @@
+  <script language="javascript" type="text/javascript" src="SocialDistancingSimulator.js"></script>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="container">
+  <h1>'Social distancing' Simulator</h1>
+
+  <p>Mit diesem Simulator kannst du den Effekt von 'social distancing' auf die
+    Verbreitung einer ansteckenden Krankheit interaktiv austesten.
+    Jede Kugel representiert eine Person, welche entweder gesund (Weiss), immun (Gelb),
+    krank (Rot) oder tot (Grau) ist.
+    Eine gesunde Person wird krank sobald sie mit einer kranken Person in Kontakt kommt.
+    Wenn eine kranke Person die Krankheit übersteht wird sie immun.
+  </p>
+
+  <p>Bewege die Regelbalken um die Parameter zu bewegen und beobachte was passiert.
+    Unter der Grafik findest du eine detailierte Beschreibung.</p>
+
+  <table class='parameters'>
+  <tr>
+  <td>'Social distance' (Distanz)</td>
+  <td><input id="social-distance-slider" class="slider" type="range" min="0" max="100" value="10"
+     oninput="document.getElementById('social-distance-value').innerHTML=this.value;"
+     onchange="document.getElementById('social-distance-value').innerHTML=this.value;"/></td>
+  <td><span id="social-distance-value">10</span>%</td>
+  <td class="restart-button" rowspan="3"><input id='restart-button' type='submit' value='Restart'/></td>
+  </tr>
+  <tr>
+  <td>Sterblichkeit</td>
+  <td><input id="mortality-slider" class="slider" type="range" min="0" max="100" value="3"
+       oninput="document.getElementById('mortality-value').innerHTML=this.value;"
+       onchange="document.getElementById('mortality-value').innerHTML=this.value;"/></td>
+  <td><span id="mortality-value">3</span>%</td>
+  </tr>
+  <tr>
+  <td>Krankheitsdauer</td>
+  <td><input id="sick-time-slider" class="slider" type="range" min="0" max="10" step="0.1" value="2"
+     oninput="document.getElementById('sick-time-value').innerHTML=Number(this.value).toFixed(1);"
+     onchange="document.getElementById('sick-time-value').innerHTML=Number(this.value).toFixed(1);"/></td>
+  <td><span id="sick-time-value">2.0</span>s</td>
+  </tr>
+  </table>
+
+  <div id="ball-simulation"></div>
+
+  <table class="statistics">
+    <tr>
+    <td><span style="font-size: 32px;">○</span> Gesunde</td>
+    <td><span style="font-size: 32px; color: #FFFF40">●</span> Immune</td>
+    <td><span style="font-size: 32px; color: #FF0000">●</span> Kranke</td>
+    <td><span style="font-size: 32px; color: #808080">●</span> Tote</td>
+    <td>Zeit</td>
+    </tr>
+    <tr>
+    <td><span id="healthy-stat"></span>%</td>
+    <td><span id="immune-stat"></span>%</td>
+    <td><span id="sick-stat"></span>%</td>
+    <td><span id="dead-stat"></span>%</td>
+    <td><span id="current-time"></span>s</td>
+    </tr>
+  </table>
+
+  <div id="ball-graph"></div>
+
+  <p><b>Social distancing</b> bestimmt in welchem Ausmaß sich die Personein voneinander "distanziert verhalten".
```

Comment:
Is there a German translation for "social distancing"?

Reply:
Well as I wrote in the PR text, Austrian/German media uses that English term as far as I've heard. Quickly googling gives a few results for "Soziale Distanzierung", which would be the plain translation, but it's also put in quotes in all articles I see because that would not be phrase that would easily goes over the lips.